### ISSUE TITLE
deploy_hono.sh

### ISSUE THREAD

**Issue by soheilade** [eclipse/kuksa.cloud#155]
Dear all, running deploy_hono.sh requires download of 
https://download.eclipse.org/hono/eclipse-hono-deploy-0.8.tar.gz
which does not exist, which files from [here](https://download.eclipse.org/hono/) must be replaced in the script?

**Comment by eriksven** [comment]
You could try to go for [https://download.eclipse.org/hono/eclipse-hono-deploy-0.9.tar.gz](https://download.eclipse.org/hono/eclipse-hono-deploy-0.9.tar.gz) since that could be as close as possible to the removed Eclipse Hono 0.8 release. However, one reason why we have not updated to a newer version of Eclipse Hono (they released 1.1.1 some days ago) earlier was that we also use the deployment of the InfluxDb and Grafana. Both components used to be part of the Eclipse Hono deployment in version 0.8 but have been removed with newer versions. So when you use a more recent version of Hono you need to manually InfluxDb and Grafana afterwards.

**Comment by eriksven** [comment]
If you want to with the latest version of Eclipse Hono one recommendation is to use the Helm chart provided as part of the Eclipse IoT Packages Edge2Cloud Package: https://github.com/eclipse/packages/tree/master/charts/hono

**Comment by soheilade** [comment]
thanks @eriksven , I am actually planning to bring this up in the call tomorrow. however, there is the problem that the helm chart does not launch any influxDB as well. One way to proceed with influxDB is to launch it as a separate microservice. and use the prometheus server to store hono internal statistics. Today, I discussed with Jens and to connect influxDB and hono we can either use kafka or use influxDB connector. Jens said iot packages is open to pull requests and if we want to add new requirements to the helm chart, it can be discussed with them. Lets discuss it tomorrow in the call :)

**Comment by eriksven** [comment]
#59

**Comment by eriksven** [comment]
You are right, somehow we need to launch the InfluxDB. I would go for a InfluxDb deployment that is not linked anymore with the Hono (or packages) deployment. My feeling is that this is not a general requirement/feature for the IoT packages project and we thus "only" need the InfluxDb within the Kuksa cloud. 
For the connection between the InfluxDb and Hono we created the InfluxDb Connector as part of the Kuksa cloud. It could be interesting to investigate whether we should Kafka instead for this. But to me this sounds like a separate issue.

**Comment by soheilade** [comment]
yup, it would also be great to know what is actually influxDB connector is doing and compare it with the features that kafka (reliability, caching, etc ... ) provides and see which one is a better fit.
